src/components/magicui/animated-grid-pattern.tsx

```
@@ -0,0 +1,154 @@
+  useState,
+} from "react";
+
+import { cn } from "@/lib/utils";
+
+export interface AnimatedGridPatternProps
+  extends ComponentPropsWithoutRef<"svg"> {
+  width?: number;
+  height?: number;
+  x?: number;
+  y?: number;
+  strokeDasharray?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)